roles/head_of_capability_practice.md

```diff
@@ -0,0 +1,85 @@
+# Head of Capability Practice
+
+UK-based in Birmingham, Bristol, Glasgow, Manchester, Newcastle, London & Swansea.
```

Comment:
I thought we'd moved away from Birmingham, Glasgow, Newcastle, Swansea as hubs now?

Reply:
I've updated this not because we have moved away from this strategy but because guidance is unclear currently.